backend/db/sqliteschema/migrations.go

```diff
@@ -219,4 +219,66 @@ var migrations = []string{
+		-- Address of the LND node backing up this wallet. In case lndhub, this will be the 
+		-- P2P addresses to connect to that site in the format of multiaddresses. Space separated.
+		addresses TEXT NOT NULL,
+		-- The account ID of the site. We need a previous connection to the site so the 
+		-- actual account is inserted in the accounts table when handshake.
+		account_id INTEGER REFERENCES accounts ON DELETE CASCADE NOT NULL
+	) WITHOUT ROWID;`,
+
+	// Table that stores all the tokens not yet redeemed inside a site. Although this table is relevant only
+	// for sites at the beginning, keep in mind that any regular node can be upgraded to a site.
+	`CREATE TABLE unredeemed_invite_links (
+		-- Unique token identification. Random 8 char words
+		token TEXT PRIMARY KEY,
+		-- The role the token will allow ROLE_UNSPECIFIED = 0 OWNER = 1 EDITOR = 2
+		role INT NOT NULL DEFAULT 2,
+		-- Timestamp since the token will no longer be eligible to be redeemed. Seconds since  Jan 1, 1970
+		expiration_time INTEGER NOT NULL CHECK (expiration_time > 0)
+	) WITHOUT ROWID;`,
+
+	// Table that stores the role each account has inside a site. Although this table is relevant only
+	// for sites at the beginning, keep in mind that any regular node can be upgraded to a site.
+	`CREATE TABLE redeemed_invite_links (
+		-- The account id that has been linked to a role on this site
+		account_id INTEGER REFERENCES accounts ON DELETE CASCADE NOT NULL PRIMARY KEY,
+		-- The role the account holds ROLE_UNSPECIFIED = 0 | OWNER = 1 | EDITOR = 2
+		role INT NOT NULL
+	) WITHOUT ROWID;`,
+
+	// Stores all the records published on this site. Although this table is relevant only
+	// for sites at the beginning, keep in mind that any regular node can be upgraded to a site.
+	`CREATE TABLE web_publication_records (
+		-- Unique identifier
+		id INTEGER PRIMARY KEY,
+		-- doc id of the base document published. Not its references.
+		document_id INTEGER REFERENCES ipfs_blocks ON DELETE CASCADE NOT NULL,
+		-- doc version of the base document published. Not its references.
+		document_version TEXT NOT NULL,
+		-- Path this publication is published. If NULL then its not pinned. If / is root document.
+		path TEXT
+		
+	);`,
+
+	// Stores all the references a published record has. When the user publishes a document it may contain
+	// links and transcluded documents. Those documents must be stored as well to properly render the base document.
+	// for sites at the beginning, keep in mind that any regular node can be upgraded to a site.
+	`CREATE TABLE web_publication_references (
+		-- Unique identifier
+		id INTEGER PRIMARY KEY,
+		-- doc id of the referenced document.
+		document_id INTEGER REFERENCES ipfs_blocks ON DELETE CASCADE NOT NULL,
+		-- doc version of the reference document.
+		document_version TEXT NOT NULL,
+		-- the published document this document is referenced in.
+		source_document_id INTEGER REFERENCES web_publication_records(id) ON DELETE CASCADE NOT NULL
+	);`,
```

Comment:
Can we use the backlinks table for that + the existing documents table?

Reply:
Ok, @burdiyan so you are saying removing this table in favor of looking up entries in `content_links` table whose `source_document_id` matches `document_id` in `web_publication_records`? Then the lookup complexity would be a bit harder since we would have to iterate for each row in `web_publication_records` all the rows in `content_links`. Oh but I see there is an index there involving `source_document_id` so is not a big of a deal

Reply:
@burdiyan Now that I rethink, in the table `content_links` which one is the source and wich one is the target? for example I made these assumptions
1. In a link, the source document is the where the link has been written. The target is the document that link points to.
2. In a transclusion, the source document is the document that uses the transcluded content and the target document is the document where the transcluded content was initially written.
Is this ok?

Reply:
@juligasa in the `content_links` table the source is always the document where the link was found initially. Target is where the link points to.

Reply:
@burdiyan I understand the link example, but if a `document B` includes transcluded content from `document A`, does this transclusion create an entry in the table `content_links`? If so, will the `source_document_id` be `document A` or `document B`?.

Reply:
Transclusion is also a link. So, for all the rows in the `content_link` table, the source is the document where the link was discovered, and target is where the link points to. Always the same.